packages/app/components/stackbox/Stackbox.tsx

```diff
@@ -486,6 +488,10 @@ export const Stackbox = () => {
                   <DatePicker
                     dateTime={endDateTime}
                     setDateTime={(date: Date) => {
+                      const isEndTimeBeforeStartTime =
+                        date.getTime() <= startDateTime.getTime();
+
+                      setShowPastEndDateError(isEndTimeBeforeStartTime);
```

Comment:
shouldn't we only set if `isEndTimeBeforeStartTime` is `true`?

Reply:
If we don't set the `false` value the error won't get cleaned on a valid date change